### PR TITLE
GH#16954 - Add examples for route allow list

### PR DESCRIPTION
**Pull request by jboxman**
Pull this forward from OCP 3.11:

- https://github.com/openshift/openshift-docs/issues/16954

Preview: https://gh-16954--ocpdocs.netlify.app/openshift-enterprise/latest/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration

### CODE REVIEW

**Comment by lihongan** [comment]
LGTM. thank you @jboxman

File: modules/nw-route-specific-annotations.adoc [hunk 1]
```diff
@@ -56,6 +57,37 @@ Setting a server-side timeout value for passthrough routes too low can cause
...
+
+.A route that allows an IP address CIDR network
+[source,yaml]
+----
+metadata:
+  annotations:
+    haproxy.router.openshift.io/ip_whitelist: 192.168.1.0/24
+----
+
+.A route that allows both IP addresses and IP address CIDR networks
```
Comment (lbarbeevargas) [inline modules/nw-route-specific-annotations.adoc:84]: This is a small nit, but in the example, there is only one IP address. The title says "IP addresses." Could you change the title to "A route that allows both an IP address and IP address CIDR networks?" Or keep the title the same and add a second IP address to the example?

**Review by lbarbeevargas — COMMENTED** [review]
@jboxman Just one small suggestion, but otherwise LGTM!

**Pull request merged**